packages/core/src/external/carequality/ihe-gateway-v2/outbound/xca/process/dq-response.ts

```diff
@@ -77,10 +77,15 @@ function parseDocumentReference(
       (c: Classification) => c._classificationScheme === classificationScheme
     );
     if (!classification) return undefined;
-    const classificationSlots = Array.isArray(classification.Slot)
-      ? classification.Slot
-      : [classification.Slot];
 
+    let classificationSlots: Slot[] = [];
+    if (Array.isArray(classification.Slot)) {
+      classificationSlots = classification.Slot.filter((slot: Slot) => slot != undefined);
+    } else if (classification.Slot) {
+      classificationSlots = [classification.Slot];
+    }
+
+    if (!classificationSlots) return undefined;
```

Comment:
`classificationSlots` is always an array from what I can see, which will always evaluate to true even when empty in TS, so not sure what this is for (i.e. I don't think it'll even get called).